azure-sql/database/performance-guidance.md

```diff
@@ -261,7 +261,15 @@ Connection pooling can improve throughput, reduce latency, and enhance the overa
    - [JDBC connection pooling](/sql/connect/jdbc/using-connection-pooling?view=azuresqldb-current&preserve-view=true)
    - [PHP connection pooling](/sql/connect/php/connection-pooling-microsoft-drivers-for-php-for-sql-server?view=azuresqldb-current&preserve-view=true)
 
-- Cloud applications should implement [retry logic](develop-overview.md#resiliency) to handle transient connectivity failures gracefully. Learn more about how to design [retry logic for transient errors](troubleshoot-common-connectivity-issues.md#retry-logic-for-transient-errors). 
+- Cloud applications should implement [retry logic](develop-overview.md#resiliency) to handle transient connectivity failures gracefully. Learn more about how to design [retry logic for transient errors](troubleshoot-common-connectivity-issues.md#retry-logic-for-transient-errors).
+
+- Token-based authentication mechanisms, such as Microsoft Entra ID authentication, may affect connection pooling due to token expiration, which can invalidate connections and interrupt reuse. To mitigate these issues, ensure tokens are refreshed proactively, configure token lifetimes to align with expected connection durations, monitor for expiration errors to trigger reauthentication, and minimize connection churn through effective pooling configurations.
```

Comment:
Many statements in this suggestion appears to be lifted from [wikipedia](https://en.wikipedia.org/wiki/Connection_pool#Authentication_Methods). I understand you may have contributed some of this same content to wikipedia, but Microsoft Docs will not be publishing content that significantly resembles wikipedia. Please rewrite in your own original words before we move forward. 

I also have concerns that many of these statements are overly generic or vague and do not provide helpful, actionable guidance.

Reply:
You are write. I was the one who wrote this in wikipedia to raise awareness about these issue. Ok. Let me check what can I do.